src/v0/destinations/braze/transform.js

```diff
@@ -155,8 +155,8 @@ function getUserAttributesObject(message, mappingJson, destination) {
     let value = get(traits, mappingJson[destKey]);
     if (value || (value === null && reservedKeys.includes(destKey))) {
       // if email is not string remove it from attributes
-      if (destKey === 'email' && typeof value !== 'string') {
-        throw new InstrumentationError('Invalid email, email must be a valid string');
+      if (destKey === 'email' && Number.isInteger(value)) {
```

Comment:
```js
switch (destKey) {
        case 'gender':
          value = formatGender(value);
          break;
        case 'email':
          if (typeof value !== 'string') {
            value = value.toString().toLowerCase();
          } else if (value !== undefined && value !== null){
            throw new InstrumentationError('Invalid email, email must be a valid string');
          }
          break;
        default:
          break;
      }
```
What about this?

Reply:
if v is null ..then our first check v!== 'string' will be true ...we will never go to else if condition

Reply:
corrected now

Reply:
updated